components/feature/app-links/src/test/java/mozilla/components/feature/app/links/AppLinksFeatureTest.kt

```diff
@@ -154,4 +155,13 @@ class AppLinksFeatureTest {
         verify(mockDialog).show(eq(mockFragmentManager), anyString())
         verify(mockOpenRedirect, never()).invoke(any(), anyBoolean(), any())
     }
+
+    @Test
+    fun `reused redirect dialog if exists`() {
```

Comment:
Hmm this doesn't test the logic that's being added here i.e. it doesn't test the new callback?

Reply:
I'll test for the new callback.  Thanks,